topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -119,6 +130,7 @@ In principle, this tutorial can be followed with any type of data provided that
 > 
 >    ```
 >    https://zenodo.org/records/15281843/files/images_and_seeds.zip
+>    https://zenodo.org/records/15424465/files/image_and_seed.zip
```

Comment:
@shiltemann do you think this is ok? Even when the top-annotation is just one Zenodo link? We could also repackage it, but my understanding is that we will redo the inputs either way in a few weeks.

Reply:
What do you mean by redo the inputs?